api/grpcserver/v2alpha1/reward_test.go

```diff
@@ -71,6 +71,7 @@ func TestRewardService_List(t *testing.T) {
 		list, err := client.List(ctx, &spacemeshv2alpha1.RewardRequest{Limit: 100})
 		require.NoError(t, err)
 		require.Len(t, rwds, len(list.Rewards))
+		require.Equal(t, len(rwds), int(list.Total))
```

Comment:
It would be good to also check if the returned `total` is really the total, not just the length of the returned list (i.e. if modifiers were removed for counting). I think that the current test would still pass if `ops.Modifiers = nil` was removed.

Reply:
Added check for total count when using limit & offset.